.github/renovate.json

```
@@ -0,0 +1,10 @@
+{
+  "extends": ["config:base"],
+  "packageRules": [
+    {
+      "matchUpdateTypes": ["minor", "patch"],
+      "matchCurrentVersion": "!/^0/",
+      "automerge": true
+    }
+  ]
```

```suggestion
  ],
  "ignoreTests": true
```

> ### Absence of tests
> By default, Renovate will not automerge until it sees passing status checks / check runs for the branch. If you have no tests but still want Renovate to automerge, you need to add "ignoreTests": true to your configuration. However, we strongly recommend you have tests in any project where you are regularly updating dependencies.

https://docs.renovatebot.com/key-concepts/automerge/